misc/ls-extensions/modules/bal-shell-service/src/test/resources/testcases/getResult/mime_types.json

```diff
@@ -80,7 +80,7 @@
     "source": "t",
     "result": {
       "shellValue": {
-        "value": "[{\"username\":\"John\",\"salary\":100,\"fullname\":{\"firstname\":\"John\",\"lastname\":\"Doe\"}},{\"username\":\"Adam\",\"salary\":300,\"fullname\":{\"firstname\":\"Adam\",\"lastname\":\"Smith\"}},{\"username\":\"Jake\",\"salary\":100,\"fullname\":{\"firstname\":\"jake\",\"lastname\":\"Peralta\"}}]",
+        "value": "[{\"username\":\"John\", \"salary\":100, \"fullname\":{\"firstname\":\"John\", \"lastname\":\"Doe\"}}, {\"username\":\"Adam\", \"salary\":300, \"fullname\":{\"firstname\":\"Adam\", \"lastname\":\"Smith\"}}, {\"username\":\"Jake\", \"salary\":100, \"fullname\":{\"firstname\":\"jake\", \"lastname\":\"Peralta\"}}]",
```

Comment:
Why we need to change these?

Reply:
The string value of `value` is generated by using this `getJsonString()` method,  with the previous implementation it does not add a space to separate values, but the new one does.
Since the generated value is still JSON parse-able it still satisfies the use case in `bal-shell-service`

Reply:
This is the expected behavior. Because with this fix we convert the record value to json before getting the json string representation. So the record value will be converted to a `map<json>` value. The json-string of a map value contains spaces between fields. 

The same behavior can be seen in the langlib method `toJsonString()` as well.